README.md

```diff
@@ -1,17 +1,21 @@
-- The packages in [azure-sdk-for-js](https://github.com/azure/azure-sdk-for-js/tree/master/sdk) are isomorphic (can be run in the browser and on the server (node.js)) with accurate TypeScript type definitions.
-- They will be actively supported moving forward.
-- You will find any equivalent JS package in `@azure` org on npm for the existing node sdk package. For example: [azure-arm-compute](https://www.npmjs.com/package/azure-arm-compute) node sdk's equivalent JS sdk is [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute). 
-- _All_ the management plane sdks for the V1 Azure API (Azure Service Management) `azure-asm-*` will still be maintained from this repository. We would expect customers using these packages to move to Azure Resource Manager API packages at the earliest.
+We are excited to announce that we are building a new [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js), which includes both Node.js and Browser support! 
 
-**NOTE:** Apart from the noticable changes in **require/import** statements, we have tried our level best to make _lesser_ breaking changes in the API calls.
+The following will help you update your application to the new SDK and related packages.
 
-### Migration
-You can find more information on migrating your scripts from **node** sdk to **JS** sdk over [here](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Migration.md).
+- **Packages**: 
+    - All of the management plane SDKs (`azure-arm-*`) and most of the data plane SDKs (`azure-*`), along with the rollup package [azure](https://www.npmjs.com/package/azure) in this repository, are deprecated as of **July, 2019**. They have been replaced by the [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) packages.
+    - The new actively managed packages in [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) support both Node.js and the Browser and ship with TypeScript type definitions.
+    - You will find all equivalent JavaScript packages in `@azure` org on npm for the existing Node.js SDK package. For example, [azure-arm-compute](https://www.npmjs.com/package/azure-arm-compute) Node.js SDK's equivalent JavaScript SDK is [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute). 
+    - All of the management plane SDKs for the V1 Azure API (Azure Service Management) `azure-asm-*` will continue to be maintained from the Azure SDK for Node.js repository. We expect customers using these packages to move to Azure Resource Manager API packages at their earliest convenience.
+- **Breaking Changes**: Apart from the noticable changes in **require/import** statements, we have tried our best to make minimal breaking changes in the API calls.
```

Comment:
While this statement is true for management libraries, it is not true for the data plane libraries. Examples are key vault, storage and service bus.

I would suggest to remove this line altogether